hazelcast/include/hazelcast/client/Member.h

```diff
@@ -48,12 +49,11 @@ namespace hazelcast {
 
             Member();
 
-            Member(const Address &address, const std::string &uuid, bool lite,
-                   const std::unordered_map<std::string, std::string> &attr);
+            Member(Address address, boost::uuids::uuid uuid, bool lite, std::unordered_map<std::string, std::string> attr);
```

Comment:
Are we not using the const& anymore in the codebase to avoid copying staff?

Reply:
this way allows both copy and move. If called by an rvalue it will be a move. if called by lvalue it will be copied. I try calling with rvalue where possible.